### ISSUE TITLE
Allow specifying custom CSS to be loaded

### ISSUE THREAD

**Issue by PatrickLerner** [jakobhellermann/wasm-server-runner#24]
I am messing around a bit with bevy and wasm and wanted to get the canvas full screen. Bevy seems to (nowadays) natively support this by just passing `fit_canvas_to_parent: true` to the `WindowDescriptor` resource. The issue is that it is not full screen height and only width, because a bit of CSS is needed to be injected into HTML:

```
body, html { height: 100%; }
```

I don't see a way to do this easily with this runner, but I think it would be great if it was easily possible.

It can be solved by adding a JavaScript snippet akin to the example and creating a function there:

```
export function set_styles() {
  const css = 'body, html { height: 100%; }';
  const head = document.head || document.getElementsByTagName('head')[0];
  const style = document.createElement('style');
  head.appendChild(style);
  style.type = 'text/css';
  style.appendChild(document.createTextNode(css));
}
```

and then running that in your main like

```
    #[cfg(target_family = "wasm")]
    snippet::set_styles();
```

but it is not the prettiest solution.

**Comment by bit-garden** [comment]
@jakobhellermann Right. This particular case was an issue I once had. Currently I am running into an issue where touch events are being normally interpreted  by the browser, however this interferes with Bevy's touch events. Things like the egui inspector for example can't be dragged around. 

I got around this in a prior setup by setting touch-events:none to stop the default touch event handling.
